web/src/components/SubscriptionLogo.jsx

```diff
@@ -0,0 +1,7 @@
+import subscribeIcon from "../assets/subscribeIcon.svg?url";
```

Comment:
It's strange that it has 'url' on the end like this. I've not seen that before but I assume it's part of this repo?